apps/src/code-studio/pd/form_components/QuestionsTable.jsx

```diff
@@ -97,7 +100,7 @@ export default class QuestionsTable extends React.Component {
     };
 
     return (
-      <FormGroup>
+      <FormGroup bsClass={this.props.className}>
```

Comment:
for my own learning, what is the prop `bsClass`?

Reply:
bootstrap class! I needed a way to target just the inputs in this QuestionsTable component that was built with bootstrap components